ansible/playbooks/kolide-fleet.yml

```diff
@@ -67,3 +67,5 @@
   - name: Start fleet
     tags: fleet
     command: nohup /usr/local/bin/fleet serve --config /root/.kolide.yml > /var/log/fleet.log 2> /var/log/fleet.err < /dev/null &
```

Comment:
This won't restart if the instance restarts. We should wrap it in an init service.

Reply:
I put a command in rc.local in lieu of a true init service, for now.